documentation/modules/ROOT/pages/index.adoc

```diff
@@ -1,28 +1,43 @@
-= Solution Patterns: Template Name
+= Building Apps around the Event Mesh
 :page-layout: home
 :sectnums:
 :sectlinks:
 :doctype: book
 
-A brief introduction of this solution pattern.
+Understanding how to structure an application can be challenging. Different
+architectural patterns often lack insight into how to build day-to-day solutions
+that are often at least as complex as the business they operate in. 
 
-_Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum_
+This solution pattern is written to make it obvious and to show simple, yet
+elegant, correct, and comprehensive way of building software, either in
+greenfield or even in legacy projects. This solution leverages the
+https://www.redhat.com/en/technologies/cloud-computing/openshift/serverless[Red Hat Serverless]
+Eventing component, which implements the https://www.redhat.com/en/topics/integration/what-is-an-event-mesh[Event Mesh] pattern.
```

Comment:
while this is a more marketing oriented link, we might also point (additionally) to here: https://knative.dev/docs/eventing/event-mesh/